docs/computing/mpip.md

```diff
@@ -0,0 +1,198 @@
+# mpiP: Lightweight MPI Profiler
+
+[mpiP](http://mpip.sourceforge.net/) is a lightweight MPI profiler which uses statistical sampling to record profiling data. At the momenmt is only available on Mahti. It generates less overhead and much less data than tracing tools. In order to use no code changes are require, only a re-link is needed. For both Mahti and Puhti, in addition to the required modules, one has the load the mpiP module: 
+```
+module load mpip
+```
+Next the code is compiled as normal, but with addition of the `-g` flag and the link to the libraries. For Mahti one should use:
+```
+-lmpiP -lm -lbfd -liberty -L<path-to-unwind-lib>-lunwind
+```
+The path to the unwind library on mahti is `/appl/spack/v014/install-tree/gcc-9.3.0/libunwind-1.3.1-otflii/lib`. 
+Similarly for Puhti the relink is done with :
+```
+-lmpiP -lm -L<path-to-iberty-lib>  -liberty -L<path-to-unwind-lib> -lunwind
+```
+The path to the iberty library is `/appl/spack/install-tree/intel-19.0.4/libiberty-2.31.1-o4es74/lib/`, while the path to unwind library is `/appl/spack/install-tree/intel-19.0.4/libunwind-1.2.1-45uplb/lib/`. 
```

Comment:
Will this work also work with the gcc stack? or does there need to be separate  mpip installation for that stack.
If it works for both it could be better to create a module which sets the proper paths and library flags, 
explicitly depending on a specific software installation in the documentation should be avoided.

Reply:
This part has to be solved. I put the intel because that is the recommended one on Puhti, but the gcc should work as well. I can see in that the folders are there in the install tree in the gcc folder. The path to the additional libraries should be loaded automatically depending on which modules are loaded.